### PR TITLE
feat(connector): [Adyen] Implement Twint in Wallets

### PR DESCRIPTION
**Pull request by Sakilmostak**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

### Additional Changes

- [x] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

```
    "payment_method": "wallet",
    "payment_method_type": "twint",
    "payment_method_data": {
        "wallet": {
            "twint_redirect": {}
        }
    },
```

## Motivation and Context

Twint was not available as a payment method so I add it.

## How did you test it?

DB775A6D-2310-4527-9020-1D5D764529AD_4_5005_c
Screen Shot 2023-06-27 at 2 35 42 PM
Screen Shot 2023-06-27 at 2 36 22 PM
Screen Shot 2023-06-27 at 2 36 30 PM
Screen Shot 2023-06-27 at 2 36 57 PM

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed submitted code
- [x] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Review by srujanchikke — APPROVED** [review]

File: crates/router/tests/connectors/adyen_uk_ui.rs [hunk 1]
```diff
@@ -84,6 +84,43 @@ async fn should_make_adyen_klarna_mandate_payment(c: WebDriver) -> Result<(), We
     Ok(())
 }
 
+async fn should_make_adyen_clearpay_payment(c: WebDriver) -> Result<(), WebDriverError> {
```
Comment (SanchithHegde) [inline crates/router/tests/connectors/adyen_uk_ui.rs:87]: Please avoid using single character variable names.

File: crates/router/tests/connectors/adyen_uk_ui.rs [hunk 2]
```diff
@@ -84,6 +84,43 @@ async fn should_make_adyen_klarna_mandate_payment(c: WebDriver) -> Result<(), We
...
+                Selector::QueryParamStr,
+                vec!["status=succeeded"],
+            )),
+        ],
+    )
+    .await?;
+    Ok(())
+}
+
+async fn should_make_adyen_twint_payment(c: WebDriver) -> Result<(), WebDriverError> {
```
Comment (SanchithHegde) [inline crates/router/tests/connectors/adyen_uk_ui.rs:104]: Same here.

File: crates/router/tests/connectors/selenium.rs [hunk 3]
```diff
@@ -398,6 +398,53 @@ pub trait SeleniumTest {
         pypl_actions.extend(actions);
         self.complete_actions(&c, pypl_actions).await
     }
+    async fn make_clearpay_payment(
+        &self,
+        c: WebDriver,
```
Comment (SanchithHegde) [inline crates/router/tests/connectors/selenium.rs:403]: Same here.

**Review by SanchithHegde — COMMENTED** [review]
Other than that, looks good to me!

File: crates/router/tests/connectors/adyen_uk_ui.rs (hunk 1 above)
Comment (Sakilmostak) [reply crates/router/tests/connectors/adyen_uk_ui.rs]: Done👍

**Review by SanchithHegde — APPROVED** [review]

**Review by SanchithHegde — DISMISSED** [review]

**Review by srujanchikke — APPROVED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by jagan-jaya — APPROVED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by ArjunKarthik — DISMISSED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Pull request merged**